### PR TITLE
Xpressive plugin - updated plugin artwork, updated the formula in the…

### PR DESCRIPTION
**Pull request by gnudles**
… presets to use integrate(f) instead of t*f, as integrate operation is more robust to frequency changes. and renamed the plugin to Xpressive instead of X-Pressive

### CODE REVIEW

**Comment by gnudles** [comment]
Screenshot from 2019-10-02 22-09-37
I'm not the greatest artist, but made it look a bit better.

**Comment by gnudles** [comment]
does anyone have a clue why it fails?

**Comment by PhysSong** [comment]
The build error is not related to your code.

File: plugins/Xpressive/Xpressive.cpp [hunk 1]
```diff
@@ -53,7 +53,7 @@
 extern "C" {
 
 Plugin::Descriptor PLUGIN_EXPORT xpressive_plugin_descriptor = { STRINGIFY(
-		PLUGIN_NAME), "X-Pressive", QT_TRANSLATE_NOOP("pluginBrowser",
+        PLUGIN_NAME), "Xpressive", QT_TRANSLATE_NOOP("pluginBrowser",
```
Comment (PhysSong) [inline plugins/Xpressive/Xpressive.cpp]: Spaces.

Comment (gnudles) [reply plugins/Xpressive/Xpressive.cpp]: right... spaces

**Comment by gnudles** [comment]
let's keep this pull request open for a couple of days to see if anyone may suggest further improvements to this plug-in

**Comment by RoxasKH** [comment]
I really like the fresh feel you give to it with this new artwork, the old was like: "why pink!?"
I especially love the new logo, but imo there's too space left between the 2 letter "S", you should cut it out a bit. Also i'd make the white outcircle of knobs on the right a little bit thinner and give a little bit of more contrast to the waveforms with the bg.
Just another question: there's no way to change also the bar between the graph and the box were you write the formulas? Because those gradients are too much '90s.
Anyway good work

**Comment by gnudles** [comment]
> I really like the fresh feel you give to it with this new artwork, the old was like: "why pink!?"
Thanks. 
> I especially love the new logo, but imo there's too space left between the 2 letter "S", you should cut it out a bit. Also i'd make the white outcircle of knobs on the right a little bit thinner and give a little bit of more contrast to the waveforms with the bg.
Wow, now I notice the distance between the two S. I think the white outline is Ok. what do you mean about the contrast in the waveforms?
> Just another question: there's no way to change also the bar between the graph and the box were you write the formulas? Because those gradients are too much '90s.
I was about to do so.
> Anyway good work
thanks pal.

**Comment by gnudles** [comment]
Maybe someone wants to add sample xpf files?

**Comment by RoxasKH** [comment]
I won't discuss about the outcircle, because they are tastes and it's good either this way. For the logo and the waveforms, you can see all here
xpressive

As you can see there's more contrast with the backround were the waveform icons are, so they are more clearly visible.

**Comment by gnudles** [comment]
Oh I see. ok!

**Comment by zonkmachine** [comment]
Looks good! @gnudles Do you want to see this merged now or are there any more changes coming?

**Comment by gnudles** [comment]
wait, I have some changes

**Comment by gnudles** [comment]
If you think it's ok now, you can merge. I was just thinking about hiding the smooth knob background when it is not active.

**Comment by zonkmachine** [comment]
> I was just thinking about hiding the smooth knob background when it is not active.

I went ahead with a merge as we have a whole lot of open PR's right now.
PS. Great work! :tractor:

**Pull request merged**